### DISCUSSION TITLE
Hacktoberfest topic

### DISCUSSION THREAD

**Discussion by rluvaton** [outline/outline#2623] [General]
It is that time again. hacktoberfest is totally opt in now:

> A repository/project is considered to be participating in Hacktoberfest if the **`hacktoberfest` topic is present** and is accepting public contributions via pull requests. An individual pull request can also be opted-in directly by adding the `hacktoberfest-accepted` label.

Could you please add the `hacktoberfest` topic?

**Comment by matbrgz** [comment]
@tommoor could you please update this for us?

**Reply by tommoor** [reply]
Done
